--- moto-2.3.2/tests/test_core/test_mock_regions.py.orig	2022-01-16 00:21:41.000000000 +0100
+++ moto-2.3.2/tests/test_core/test_mock_regions.py	2023-12-09 12:03:22.924853278 +0100
@@ -1,9 +1,8 @@
 import boto3
-import mock
 import os
 import pytest
 from moto import mock_sns, settings
-from unittest import SkipTest
+from unittest import SkipTest, mock
 
 
 @mock_sns
--- moto-2.3.2/tests/test_ecs/test_ecs_boto3.py.orig	2022-01-16 00:21:41.000000000 +0100
+++ moto-2.3.2/tests/test_ecs/test_ecs_boto3.py	2023-12-09 12:04:54.281025026 +0100
@@ -2,7 +2,6 @@ from datetime import datetime
 
 from botocore.exceptions import ClientError
 import boto3
-import mock
 import sure  # noqa # pylint: disable=unused-import
 import json
 import os
@@ -21,7 +20,7 @@ from moto.ecs.exceptions import (
 )
 import pytest
 from tests import EXAMPLE_AMI_ID
-from unittest import SkipTest
+from unittest import SkipTest, mock
 
 
 @mock_ecs
--- moto-2.3.2/tests/test_eks/test_eks.py.orig	2022-01-16 00:21:41.000000000 +0100
+++ moto-2.3.2/tests/test_eks/test_eks.py	2023-12-09 12:05:55.744025386 +0100
@@ -1,8 +1,7 @@
 from copy import deepcopy
-from unittest import SkipTest
+from unittest import SkipTest, mock
 
 import boto3
-import mock
 import pytest
 import sure  # noqa # pylint: disable=unused-import
 from botocore.exceptions import ClientError
