--- moto-2.1.0/setup.py.orig	2023-11-14 22:40:42.204978540 +0100
+++ moto-2.1.0/setup.py	2023-11-15 21:06:33.204179502 +0100
@@ -64,9 +64,9 @@ install_requires += [
     "more-itertools",
     "more-itertools==5.0.0; python_version < '3'",
     # Indirect - Py2 works with 4.5, breaks with 4.7, but officially only supported by 4.0
-    "rsa<=4.0; python_version < '3'",
+    "rsa<=4.5; python_version < '3'",
     "setuptools",
-    "setuptools==44.0.0; python_version < '3'",
+    "setuptools~=44.0; python_version < '3'",
 ]
 
 _dep_PyYAML = "PyYAML>=5.1"
@@ -78,7 +78,7 @@ _dep_python_jose_ecdsa_pin = (
 _dep_docker = "docker>=2.5.1"
 _dep_jsondiff = "jsondiff>=1.1.2"
 _dep_aws_xray_sdk = "aws-xray-sdk!=0.96,>=0.93"
-_dep_idna = "idna<3,>=2.5"
+_dep_idna = "idna<4,>=2.5"
 _dep_cfn_lint = "cfn-lint>=0.4.0"
 _dep_decorator = "decorator<=4.4.2; python_version<'3'"  # Transitive dependency - last version that supports py2.7
 _dep_sshpubkeys_py2 = "sshpubkeys==3.1.0; python_version<'3'"
